velox/docs/.mailmap

```diff
@@ -0,0 +1,54 @@
+Amit Dutta <Meta> <adutta@meta.com>
+Andrii Rosa <Meta> <andriirosa@fb.com>
+Andrii Rosa <Meta> <andriirosa@meta.com>
+Ankita Victor <Microsoft> <anvicto@microsoft.com>
+Athmaja N <IBM> <Athmaja.N@ibm.com>
+Bikramjeet Vig <Meta> <bikramjeet@meta.com>
+Daniel Hunte <Meta> <danielhunte@meta.com>
+Daniel Jonathan Hunte <Meta> <danielhunte@meta.com>
+Deepak Majeti <IBM> <deepak.majeti@ibm.com>
+Deepashree Gandhi <IBM> <deegandh@in.ibm.com>
+Huameng (Michael) Jiang <Meta> <huamengjiang@meta.com>
+Jacob Khaliqi <IBM> <jacobkhaliqi@gmail.com>
+Jacob Wujciak-Jens <Voltron Data> <jacob@wujciak.de>
+Jia Ke <Intel> <ke.a.jia@intel.com>
+Jialiang Tan <Meta> <jtan6@meta.com>
+Jialiang Tan <Meta> <jacob.jialiang.tan@gmail.com>
+Jimmy Lu <Meta> <jimmylu@meta.com>
+Kevin Wilfong <Meta> <kevinwilfong@fb.com>
+Kevin Wilfong <Meta> <kevinwilfong@meta.com>
+Krishna Pai <Meta> <kpai@fb.com>
+Kunal Kataria <Meta> <kunalkataria@meta.com>
+Masha Basmanova <Meta> <mbasmanova@fb.com>
+Masha Basmanova <Meta> <mbasmanova@meta.com>
+Orri Erling <Meta> <oerling@fb.com>
+PHILO-HE <Intel> <feilong.he@intel.com>
+Pedro Eugenio Rocha Pedreira <Meta> <pedroerp@meta.com>
+Pedro Pedreira <Meta> <pedroerp@meta.com>
+Pranjal Shankhdhar <Meta> <pranjalssh@meta.com>
+Pramod <IBM> <pramod@ahana.io>
```

Comment:
Thanks @majetideepak. Could you please update my name to the preferred name `Pramod Satya`, and also add the email address `pramod.satya@ibm.com`? I recently updated my Github profile to reflect the same. 
```
Pramod Satya <IBM> <pramod@ahana.io>
Pramod Satya <IBM> <pramod.satya@ibm.com>
```